### ISSUE TITLE
Client update required

### ISSUE THREAD

**Issue by surajsub** [IBM-Cloud/bluemix-go#221]
When  a resource instance for hpvs is required to be created via terraform it need a sshkey to be passed in . The current client structure does not permit that 
CreateServiceInstanceRequest struct {
	Name            string                 `json:"name"`
	ServicePlanID   string                 `json:"resource_plan_id"`
	ResourceGroupID string                 `json:"resource_group_id"`
	Crn             string                 `json:"crn,omitempty"`
	Tags            []string               `json:"tags,omitempty"`
	Parameters      map[string]interface{} `json:"parameters,omitempty"`
	TargetCrn       string                 `json:"target_crn"`
}
If we are to create the hpvs via terraform this structure needs to be update to add the SSHKey as well.

**Comment by hkantare** [comment]
Can you let me know how you provision the same service using the ibmcloud  resource cli command..I think the ssh key needs to be passed as parameter

**Comment by hkantare** [comment]
The API is designed as per API doc https://cloud.ibm.com/apidocs/resource-controller/resource-controller#create-provision-a-new-resource-instance

**Comment by surajsub** [comment]
Yes - that API specification is incomplete. If you need to provision a HPVS then you need to pass in the key.. We can make it an optional parameter.

**Comment by hkantare** [comment]
We need to pass the key as a value in parameter....Add the key(name and value ) in Parameters
